omrtrace/omrtrace_internal.h

```diff
@@ -20,43 +20,35 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-#ifndef OMR_TRACE_INTERNAL_H_INCLUDED
+#if !defined(OMR_TRACE_INTERNAL_H_INCLUDED)
 #define OMR_TRACE_INTERNAL_H_INCLUDED
 
 #include "omrpool.h"
 #include "omrtrace.h"
 #include "ute_core.h"
 #include "ute_dataformat.h"
 
-#ifdef  __cplusplus
+#if defined(__cplusplus)
 extern "C" {
-#endif
+#endif /* defined(__cplusplus) */
 
-
-#define OS_THREAD_FROM_TRACE_THREAD(thr)	((omrthread_t)(thr)->synonym1)
-#define OMR_VMTHREAD_FROM_TRACE_THREAD(thr)	((OMR_VMThread *)(thr)->synonym2)
+#define OS_THREAD_FROM_TRACE_THREAD(thr)    ((omrthread_t)(thr)->synonym1)
+#define OMR_VMTHREAD_FROM_TRACE_THREAD(thr) ((OMR_VMThread *)(thr)->synonym2)
 #define OMR_TRACE_THREAD_FROM_ENV(env) \
-	( env? OMR_TRACE_THREAD_FROM_VMTHREAD((OMR_VMThread *)env) : twThreadSelf() )
+	((NULL != env) ? OMR_TRACE_THREAD_FROM_VMTHREAD((OMR_VMThread *)(env)) : twThreadSelf())
```

Comment:
```suggestion
	((NULL != (env)) ? OMR_TRACE_THREAD_FROM_VMTHREAD((OMR_VMThread *)(env)) : twThreadSelf())
```
Should there be parentheses around `env` (since I see parentheses later on)